rust/rbac-registration/src/cardano/cip509/validation.rs

```diff
@@ -105,7 +105,7 @@ pub fn validate_aux(
     let hash = Blake2b256Hash::new(raw_aux_data);
     if hash != auxiliary_data_hash {
         report.other(
-            &format!("Incorrect transaction auxiliary data hash = '{hash:?}', expected = '{auxiliary_data_hash:?}'"),
+            &format!("Incorrect transaction auxiliary data hash = '{hash}', expected = '{auxiliary_data_hash}'"),
```

Comment:
```suggestion
            &format!("Incorrect transaction auxiliary data hash = '0x{hash}', expected = '0x{auxiliary_data_hash}'.  This metadata does not belong with this Transaction.  Catalyst Metadata may not be transcribed to any other transaction body, and is therefore invalid."),
```

Again, don't just say What went wrong, also say why its important.